// @clerk/localizations/dist/pl-PL.js
"use strict";
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);

// src/pl-PL.ts
var pl_PL_exports = {};
__export(pl_PL_exports, {
  plPL: () => plPL
});
module.exports = __toCommonJS(pl_PL_exports);
var plPL = {
  locale: "pl-PL",
  backButton: "Powr\xF3t",
  badge__default: "Domy\u015Blny",
  badge__otherImpersonatorDevice: "Inne urz\u0105dzenie osobiste",
  badge__primary: "Podstawowy",
  badge__requiresAction: "Wymaga dzia\u0142ania",
  badge__thisDevice: "To urz\u0105dzenie",
  badge__unverified: "Niezweryfikowany",
  badge__userDevice: "Urz\u0105dzenie u\u017Cytkownika",
  badge__you: "Ty",
  createOrganization: {
    formButtonSubmit: "Utw\xF3rz organizacj\u0119",
    invitePage: {
      formButtonReset: "Pomi\u0144"
    },
    title: "Utw\xF3rz organizacj\u0119"
  },
  dates: {
    lastDay: "Wczoraj o godzinie {{ date | timeString('pl-PL') }}",
    next6Days: "{{ date | weekday('pl-PL','long') }} o godzinie {{ date | timeString('pl-PL') }}",
    nextDay: "Jutro o godzinie {{ date | timeString('pl-PL') }}",
    numeric: "{{ date | numeric('pl-PL') }}",
    previous6Days: "Ostatni(a) {{ date | weekday('pl-PL','long') }} o godzinie {{ date | timeString('pl-PL') }}",
    sameDay: "Dzisiaj o godzinie {{ date | timeString('pl-PL') }}"
  },
  dividerText: "lub",
  footerActionLink__useAnotherMethod: "U\u017Cyj innej metody",
  footerPageLink__help: "Pomoc",
  footerPageLink__privacy: "Prywatno\u015B\u0107",
  footerPageLink__terms: "Warunki",
  formButtonPrimary: "Kontynuuj",
  formButtonPrimary__verify: "Zweryfikuj",
  formFieldAction__forgotPassword: "Zapomnia\u0142em/am has\u0142a",
  formFieldError__matchingPasswords: "Has\u0142a si\u0119 zgadzaj\u0105.",
  formFieldError__notMatchingPasswords: "Has\u0142a si\u0119 nie zgadzaj\u0105.",
  formFieldError__verificationLinkExpired: "Link weryfikacyjny wygas\u0142. Spr\xF3buj ponownie.",
  formFieldHintText__optional: "Opcjonalne",
  formFieldHintText__slug: "Jest to unikalne ID, kt\xF3re jest czytelne dla cz\u0142owieka, cz\u0119sto u\u017Cywane w adresach URL.",
  formFieldInputPlaceholder__backupCode: void 0,
  formFieldInputPlaceholder__confirmDeletionUserAccount: "Usu\u0144 konto",
  formFieldInputPlaceholder__emailAddress: void 0,
  formFieldInputPlaceholder__emailAddress_username: void 0,
  formFieldInputPlaceholder__emailAddresses: "Wprowad\u017A lub wklej jeden lub wi\u0119cej adres\xF3w e-mail, oddzielonych spacjami lub przecinkami",
  formFieldInputPlaceholder__firstName: void 0,
  formFieldInputPlaceholder__lastName: void 0,
  formFieldInputPlaceholder__organizationDomain: void 0,
  formFieldInputPlaceholder__organizationDomainEmailAddress: void 0,
  formFieldInputPlaceholder__organizationName: void 0,
  formFieldInputPlaceholder__organizationSlug: void 0,
  formFieldInputPlaceholder__password: void 0,
  formFieldInputPlaceholder__phoneNumber: void 0,
  formFieldInputPlaceholder__username: void 0,
  formFieldLabel__automaticInvitations: "W\u0142\u0105cz automatyczne zaproszenia dla tej domeny",
  formFieldLabel__backupCode: "Kod zapasowy",
  formFieldLabel__confirmDeletion: "Potwierdzenie",
  formFieldLabel__confirmPassword: "Potwierd\u017A has\u0142o",
  formFieldLabel__currentPassword: "Obecne has\u0142o",
  formFieldLabel__emailAddress: "Adres e-mail",
  formFieldLabel__emailAddress_username: "Adres e-mail lub nazwa u\u017Cytkownika",
  formFieldLabel__emailAddresses: "Adresy e-mail",
  formFieldLabel__firstName: "Imi\u0119",
  formFieldLabel__lastName: "Nazwisko",
  formFieldLabel__newPassword: "Nowe has\u0142o",
  formFieldLabel__organizationDomain: "Domain",
  formFieldLabel__organizationDomainDeletePending: "Delete pending invitations and suggestions",
  formFieldLabel__organizationDomainEmailAddress: "Verification email address",
  formFieldLabel__organizationDomainEmailAddressDescription: "Enter an email address under this domain to receive a code and verify this domain.",
  formFieldLabel__organizationName: "Nazwa organizacji",
  formFieldLabel__organizationSlug: "Slug URL",
  formFieldLabel__passkeyName: void 0,
  formFieldLabel__password: "Has\u0142o",
  formFieldLabel__phoneNumber: "Numer telefonu",
  formFieldLabel__role: "Rola",
  formFieldLabel__signOutOfOtherSessions: "Wyloguj si\u0119 ze wszystkich innych urz\u0105dze\u0144",
  formFieldLabel__username: "Nazwa u\u017Cytkownika",
  impersonationFab: {
    action__signOut: "Wyloguj",
    title: "Zalogowano jako {{identifier}}"
  },
  maintenanceMode: void 0,
  membershipRole__admin: "Administrator",
  membershipRole__basicMember: "U\u017Cytkownik",
  membershipRole__guestMember: "Go\u015B\u0107",
  organizationList: {
    action__createOrganization: "Stw\xF3rz organizacj\u0119",
    action__invitationAccept: "Do\u0142\u0105cz",
    action__suggestionsAccept: "Popro\u015B o do\u0142\u0105czenie",
    createOrganization: "Stw\xF3rz organizacj\u0119",
    invitationAcceptedLabel: "Do\u0142\u0105czono",
    subtitle: "to continue to {{applicationName}}",
    suggestionsAcceptedLabel: "Pending approval",
    title: "Wybierz konto",
    titleWithoutPersonal: "Wybierz organizacj\u0119"
  },
  organizationProfile: {
    badge__automaticInvitation: "Automatic invitations",
    badge__automaticSuggestion: "Automatic suggestions",
    badge__manualInvitation: "No automatic enrollment",
    badge__unverified: "Unverified",
    createDomainPage: {
      subtitle: "Add the domain to verify. Users with email addresses at this domain can join the organization automatically or request to join.",
      title: "Dodaj domen\u0119"
    },
    invitePage: {
      detailsTitle__inviteFailed: "Nie uda\u0142o si\u0119 wys\u0142a\u0107 zaprosze\u0144. Napraw poni\u017Csze problemy i spr\xF3buj ponownie:",
      formButtonPrimary__continue: "Wy\u015Blij zaproszenia",
      selectDropdown__role: "Select role",
      subtitle: "Zapro\u015B nowych u\u017Cytkownik\xF3w do tej organizacji",
      successMessage: "Zaproszenia zosta\u0142y pomy\u015Blnie wys\u0142ane",
      title: "Zapro\u015B u\u017Cytkownik\xF3w"
    },
    membersPage: {
      action__invite: "Zapro\u015B",
      activeMembersTab: {
        menuAction__remove: "Usu\u0144 u\u017Cytkownika",
        tableHeader__actions: void 0,
        tableHeader__joined: "Do\u0142\u0105czy\u0142",
        tableHeader__role: "Rola",
        tableHeader__user: "U\u017Cytkownik"
      },
      detailsTitle__emptyRow: "Brak u\u017Cytkownik\xF3w do wy\u015Bwietlenia",
      invitationsTab: {
        autoInvitations: {
          headerSubtitle: "Invite users by connecting an email domain with your organization. Anyone who signs up with a matching email domain will be able to join the organization anytime.",
          headerTitle: "Automatic invitations",
          primaryButton: "Manage verified domains"
        },
        table__emptyRow: "No invitations to display"
      },
      invitedMembersTab: {
        menuAction__revoke: "Anuluj zaproszenie",
        tableHeader__invited: "Zaproszony"
      },
      requestsTab: {
        autoSuggestions: {
          headerSubtitle: "Users who sign up with a matching email domain, will be able to see a suggestion to request to join your organization.",
          headerTitle: "Automatic suggestions",
          primaryButton: "Manage verified domains"
        },
        menuAction__approve: "Zatwierd\u017A",
        menuAction__reject: "Odrzu\u0107",
        tableHeader__requested: "Requested access",
        table__emptyRow: "No requests to display"
      },
      start: {
        headerTitle__invitations: "Zaproszenia",
        headerTitle__members: "Cz\u0142onkowie",
        headerTitle__requests: "Requests"
      }
    },
    navbar: {
      description: "Zarz\u0105dzaj organizacj\u0105.",
      general: "G\u0142\xF3wne",
      members: "Cz\u0142onkowie",
      title: "Organizacja"
    },
    profilePage: {
      dangerSection: {
        deleteOrganization: {
          actionDescription: 'Type "{{organizationName}}" below to continue.',
          messageLine1: "Are you sure you want to delete this organization?",
          messageLine2: "This action is permanent and irreversible.",
          successMessage: "You have deleted the organization.",
          title: "Usu\u0144 organizacj\u0119"
        },
        leaveOrganization: {
          actionDescription: 'Type "{{organizationName}}" below to continue.',
          messageLine1: "Czy na pewno chcesz opu\u015Bci\u0107 t\u0119 organizacj\u0119? Stracisz dost\u0119p do tej organizacji i jej aplikacji.",
          messageLine2: "Ta akcja jest trwa\u0142a i nieodwracalna.",
          successMessage: "Opu\u015Bci\u0142e\u015B organizacj\u0119.",
          title: "Opu\u015B\u0107 organizacj\u0119"
        },
        title: "Zagro\u017Cenie"
      },
      domainSection: {
        menuAction__manage: "Zarz\u0105dzaj",
        menuAction__remove: "Usu\u0144",
        menuAction__verify: "Zweryfikuj",
        primaryButton: "Dodaj domen\u0119",
        subtitle: "Allow users to join the organization automatically or request to join based on a verified email domain.",
        title: "Zweryfikowane domeny"
      },
      successMessage: "Organizacja zosta\u0142a zaktualizowana.",
      title: "Profil organizacji"
    },
    removeDomainPage: {
      messageLine1: "The email domain {{domain}} will be removed.",
      messageLine2: "Users won\u2019t be able to join the organization automatically after this.",
      successMessage: "{{domain}} has been removed.",
      title: "Remove domain"
    },
    start: {
      headerTitle__general: "General",
      headerTitle__members: "Cz\u0142onkowie",
      profileSection: {
        primaryButton: void 0,
        title: "Organization Profile",
        uploadAction__title: "Logo"
      }
    },
    verifiedDomainPage: {
      dangerTab: {
        calloutInfoLabel: "Removing this domain will affect invited users.",
        removeDomainActionLabel__remove: "Usu\u0144 domen\u0119",
        removeDomainSubtitle: "Usu\u0144 t\u0105 domen\u0119 ze zweryfikowanych domen",
        removeDomainTitle: "Usu\u0144 domen\u0119"
      },
      enrollmentTab: {
        automaticInvitationOption__description: "Users are automatically invited to join the organization when they sign-up and can join anytime.",
        automaticInvitationOption__label: "Automatic invitations",
        automaticSuggestionOption__description: "Users receive a suggestion to request to join, but must be approved by an admin before they are able to join the organization.",
        automaticSuggestionOption__label: "Automatic suggestions",
        calloutInfoLabel: "Changing the enrollment mode will only affect new users.",
        calloutInvitationCountLabel: "Pending invitations sent to users: {{count}}",
        calloutSuggestionCountLabel: "Pending suggestions sent to users: {{count}}",
        manualInvitationOption__description: "Users can only be invited manually to the organization.",
        manualInvitationOption__label: "No automatic enrollment",
        subtitle: "Choose how users from this domain can join the organization."
      },
      start: {
        headerTitle__danger: "Danger",
        headerTitle__enrollment: "Enrollment options"
      },
      subtitle: "The domain {{domain}} is now verified. Continue by selecting enrollment mode.",
      title: "Update {{domain}}"
    },
    verifyDomainPage: {
      formSubtitle: "Enter the verification code sent to your email address",
      formTitle: "Kod weryfikacyjny",
      resendButton: "Didn't receive a code? Resend",
      subtitle: "The domain {{domainName}} needs to be verified via email.",
      subtitleVerificationCodeScreen: "A verification code was sent to {{emailAddress}}. Enter the code to continue.",
      title: "Zweryfikuj domen\u0119"
    }
  },
  organizationSwitcher: {
    action__createOrganization: "Utw\xF3rz organizacj\u0119",
    action__invitationAccept: "Do\u0142\u0105cz",
    action__manageOrganization: "Zarz\u0105dzaj organizacj\u0105",
    action__suggestionsAccept: "Request to join",
    notSelected: "Nie wybrano organizacji",
    personalWorkspace: "Przestrze\u0144 osobista",
    suggestionsAcceptedLabel: "Pending approval"
  },
  paginationButton__next: "Nast\u0119pny",
  paginationButton__previous: "Poprzedni",
  paginationRowText__displaying: "Wy\u015Bwietlanie",
  paginationRowText__of: "z",
  signIn: {
    accountSwitcher: {
      action__addAccount: "Dodaj konto",
      action__signOutAll: "Wyloguj si\u0119 ze wszystkich kont",
      subtitle: "Select the account with which you wish to continue.",
      title: "Wybierz konto"
    },
    alternativeMethods: {
      actionLink: "Uzyskaj pomoc",
      actionText: "Nie u\u017Cywasz \u017Cadnej z tych metod?",
      blockButton__backupCode: "U\u017Cyj kodu zapasowego",
      blockButton__emailCode: "Wy\u015Blij kod do {{identifier}}",
      blockButton__emailLink: "Wy\u015Blij link do {{identifier}}",
      blockButton__passkey: void 0,
      blockButton__password: "Zaloguj si\u0119 za pomoc\u0105 has\u0142a",
      blockButton__phoneCode: "Wy\u015Blij kod do {{identifier}}",
      blockButton__totp: "U\u017Cyj aplikacji uwierzytelniaj\u0105cej",
      getHelp: {
        blockButton__emailSupport: "Wy\u015Blij e-mail do pomocy technicznej",
        content: "Je\u015Bli masz problem z zalogowaniem si\u0119 do swojego konta, wy\u015Blij do nas e-mail, a postaramy si\u0119 jak najszybciej przywr\xF3ci\u0107 dost\u0119p.",
        title: "Uzyskaj pomoc"
      },
      subtitle: "Masz problem? Mo\u017Cesz u\u017Cy\u0107 dowolnej z tych metod weryfikacji.",
      title: "U\u017Cyj innego sposobu"
    },
    backupCodeMfa: {
      subtitle: "aby przej\u015B\u0107 do {{applicationName}}",
      title: "Wprowad\u017A kod zapasowy"
    },
    emailCode: {
      formTitle: "Kod weryfikacyjny",
      resendButton: "Wy\u015Blij kod ponownie",
      subtitle: "aby kontynuowa\u0107 w {{applicationName}}",
      title: "Sprawd\u017A swoj\u0105 poczt\u0119 e-mail"
    },
    emailLink: {
      clientMismatch: {
        subtitle: void 0,
        title: void 0
      },
      expired: {
        subtitle: "Powr\xF3\u0107 do oryginalnej karty, aby kontynuowa\u0107.",
        title: "Ten link weryfikacyjny wygas\u0142"
      },
      failed: {
        subtitle: "Powr\xF3\u0107 do oryginalnej karty, aby kontynuowa\u0107.",
        title: "Ten link weryfikacyjny jest nieprawid\u0142owy"
      },
      formSubtitle: "U\u017Cyj linku weryfikacyjnego wys\u0142anego na Tw\xF3j adres e-mail",
      formTitle: "Link weryfikacyjny",
      loading: {
        subtitle: "Zostaniesz przekierowany wkr\xF3tce",
        title: "Logowanie..."
      },
      resendButton: "Wy\u015Blij link ponownie",
      subtitle: "aby kontynuowa\u0107 w {{applicationName}}",
      title: "Sprawd\u017A swoj\u0105 poczt\u0119 e-mail",
      unusedTab: {
        title: "Mo\u017Cesz zamkn\u0105\u0107 t\u0119 kart\u0119"
      },
      verified: {
        subtitle: "Zostaniesz przekierowany wkr\xF3tce",
        title: "Pomy\u015Blnie zalogowano"
      },
      verifiedSwitchTab: {
        subtitle: "Powr\xF3\u0107 do oryginalnej karty, aby kontynuowa\u0107",
        subtitleNewTab: "Powr\xF3\u0107 do nowo otwartej karty, aby kontynuowa\u0107",
        titleNewTab: "Zalogowano na innej karcie"
      }
    },
    forgotPassword: {
      formTitle: "Kod werfikacyjny resetowania has\u0142a",
      resendButton: "Nie otrzyma\u0142e\u015B kodu? Wy\u015Blij ponownie",
      subtitle: "aby zresetowa\u0107 has\u0142o",
      subtitle_email: "Najpierw wprowad\u017A kod wys\u0142any na Tw\xF3j adres e-mail",
      subtitle_phone: "Najpierw wprowad\u017A kod wys\u0142any na Tw\xF3j numer telefonu",
      title: "Zmie\u0144 has\u0142o"
    },
    forgotPasswordAlternativeMethods: {
      blockButton__resetPassword: "Zresetuj has\u0142o",
      label__alternativeMethods: "Lub zaloguj si\u0119 za pomoc\u0105 innej metody",
      title: "Zapomnia\u0142e\u015B has\u0142a?"
    },
    noAvailableMethods: {
      message: "Nie mo\u017Cna kontynuowa\u0107 logowania. Brak dost\u0119pnych czynnik\xF3w uwierzytelniaj\u0105cych.",
      subtitle: "Wyst\u0105pi\u0142 b\u0142\u0105d",
      title: "Nie mo\u017Cna si\u0119 zalogowa\u0107"
    },
    passkey: {
      subtitle: void 0,
      title: void 0
    },
    password: {
      actionLink: "U\u017Cyj innego sposobu",
      subtitle: "aby kontynuowa\u0107 w {{applicationName}}",
      title: "Wprowad\u017A swoje has\u0142o"
    },
    passwordPwned: {
      title: void 0
    },
    phoneCode: {
      formTitle: "Kod weryfikacyjny",
      resendButton: "Wy\u015Blij kod ponownie",
      subtitle: "aby przej\u015B\u0107 do {{applicationName}}",
      title: "Sprawd\u017A sw\xF3j telefon"
    },
    phoneCodeMfa: {
      formTitle: "Kod weryfikacyjny",
      resendButton: "Wy\u015Blij kod ponownie",
      subtitle: void 0,
      title: "Sprawd\u017A sw\xF3j telefon"
    },
    resetPassword: {
      formButtonPrimary: "Zmie\u0144 has\u0142o",
      requiredMessage: "Z powod\xF3w bezpiecze\u0144stwa konieczne jest zresetowanie has\u0142a.",
      successMessage: "Twoje has\u0142o zosta\u0142o pomy\u015Blnie zresetowane. Logujemy Ci\u0119, prosz\u0119 czeka\u0107...",
      title: "Ustaw nowe has\u0142o"
    },
    resetPasswordMfa: {
      detailsLabel: "We need to verify your identity before resetting your password."
    },
    start: {
      actionLink: "Zarejestruj si\u0119",
      actionLink__join_waitlist: void 0,
      actionLink__use_email: "U\u017Cyj adresu e-mail",
      actionLink__use_email_username: "U\u017Cyj adresu e-mail lub nazwy u\u017Cytkownika",
      actionLink__use_passkey: void 0,
      actionLink__use_phone: "U\u017Cyj numeru telefonu",
      actionLink__use_username: "U\u017Cyj nazwy u\u017Cytkownika",
      actionText: "Nie masz konta?",
      actionText__join_waitlist: void 0,
      subtitle: "aby przej\u015B\u0107 do {{applicationName}}",
      title: "Zaloguj si\u0119"
    },
    totpMfa: {
      formTitle: "Kod weryfikacyjny",
      subtitle: void 0,
      title: "Weryfikacja dwustopniowa"
    }
  },
  signInEnterPasswordTitle: "Wprowad\u017A swoje has\u0142o",
  signUp: {
    continue: {
      actionLink: "Zaloguj si\u0119",
      actionText: "Masz ju\u017C konto?",
      subtitle: "aby kontynuowa\u0107 w {{applicationName}}",
      title: "Uzupe\u0142nij brakuj\u0105ce pola"
    },
    emailCode: {
      formSubtitle: "Wprowad\u017A kod weryfikacyjny wys\u0142any na Tw\xF3j adres e-mail",
      formTitle: "Kod weryfikacyjny",
      resendButton: "Wy\u015Blij ponownie",
      subtitle: "aby kontynuowa\u0107 w {{applicationName}}",
      title: "Zweryfikuj sw\xF3j adres e-mail"
    },
    emailLink: {
      clientMismatch: {
        subtitle: void 0,
        title: void 0
      },
      formSubtitle: "U\u017Cyj linku weryfikacyjnego wys\u0142anego na Tw\xF3j adres e-mail",
      formTitle: "Link weryfikacyjny",
      loading: {
        title: "Rejestrowanie..."
      },
      resendButton: "Wy\u015Blij ponownie",
      subtitle: "aby kontynuowa\u0107 w {{applicationName}}",
      title: "Zweryfikuj sw\xF3j adres e-mail",
      verified: {
        title: "Pomy\u015Blnie zarejestrowano"
      },
      verifiedSwitchTab: {
        subtitle: "Powr\xF3\u0107 do nowo otwartej karty, aby kontynuowa\u0107",
        subtitleNewTab: "Powr\xF3\u0107 do poprzedniej karty, aby kontynuowa\u0107",
        title: "Adres e-mail zosta\u0142 pomy\u015Blnie zweryfikowany"
      }
    },
    legalConsent: {
      checkbox: {
        label__onlyPrivacyPolicy: void 0,
        label__onlyTermsOfService: void 0,
        label__termsOfServiceAndPrivacyPolicy: void 0
      },
      continue: {
        subtitle: void 0,
        title: void 0
      }
    },
    phoneCode: {
      formSubtitle: "Wprowad\u017A kod weryfikacyjny wys\u0142any na Tw\xF3j numer telefonu",
      formTitle: "Kod weryfikacyjny",
      resendButton: "Wy\u015Blij ponownie",
      subtitle: "aby kontynuowa\u0107 w {{applicationName}}",
      title: "Zweryfikuj sw\xF3j numer telefonu"
    },
    restrictedAccess: {
      actionLink: void 0,
      actionText: void 0,
      blockButton__emailSupport: void 0,
      blockButton__joinWaitlist: void 0,
      subtitle: void 0,
      subtitleWaitlist: void 0,
      title: void 0
    },
    start: {
      actionLink: "Zaloguj si\u0119",
      actionLink__use_email: void 0,
      actionLink__use_phone: void 0,
      actionText: "Masz ju\u017C konto?",
      subtitle: "aby kontynuowa\u0107 w {{applicationName}}",
      title: "Utw\xF3rz swoje konto"
    }
  },
  socialButtonsBlockButton: "Kontynuuj z {{provider|titleize}}",
  socialButtonsBlockButtonManyInView: void 0,
  unstable__errors: {
    already_a_member_in_organization: void 0,
    captcha_invalid: "Rejestracja nie powiod\u0142a si\u0119 z powodu nieprawid\u0142owej weryfikacji bot\xF3w. Od\u015Bwie\u017C stron\u0119, aby spr\xF3bowa\u0107 ponownie lub skontaktuj si\u0119 z pomoc\u0105, aby uzyska\u0107 wsparcie.",
    captcha_unavailable: "Rejestracja nie powiod\u0142a si\u0119 z powodu niedost\u0119pno\u015Bci weryfikacji bot\xF3w. Od\u015Bwie\u017C stron\u0119, aby spr\xF3bowa\u0107 ponownie lub skontaktuj si\u0119 z pomoc\u0105, aby uzyska\u0107 wsparcie.",
    form_code_incorrect: void 0,
    form_identifier_exists: void 0,
    form_identifier_exists__email_address: "Adres e-mail jest ju\u017C zaj\u0119ty. Prosz\u0119 spr\xF3bowa\u0107 innego.",
    form_identifier_exists__phone_number: void 0,
    form_identifier_exists__username: void 0,
    form_identifier_not_found: void 0,
    form_param_format_invalid: void 0,
    form_param_format_invalid__email_address: "Adres e-mail musi by\u0107 poprawnym adresem e-mail.",
    form_param_format_invalid__phone_number: "Phone number must be in a valid international format",
    form_param_max_length_exceeded__first_name: "Imi\u0119 nie powinno przekracza\u0107 256 znak\xF3w.",
    form_param_max_length_exceeded__last_name: "Nazwisko nie powinno przekracza\u0107 256 znak\xF3w.",
    form_param_max_length_exceeded__name: "Nazwa nie powinna przekracza\u0107 256 znak\xF3w.",
    form_param_nil: void 0,
    form_param_value_invalid: void 0,
    form_password_incorrect: void 0,
    form_password_length_too_short: void 0,
    form_password_not_strong_enough: "Twoje has\u0142o nie jest wystarczaj\u0105co silne",
    form_password_pwned: void 0,
    form_password_pwned__sign_in: void 0,
    form_password_size_in_bytes_exceeded: "Your password has exceeded the maximum number of bytes allowed, please shorten it or remove some special characters.",
    form_password_validation_failed: "Podane has\u0142o jest nieprawid\u0142owe",
    form_username_invalid_character: void 0,
    form_username_invalid_length: void 0,
    identification_deletion_failed: "You cannot delete your last identification.",
    not_allowed_access: void 0,
    organization_domain_blocked: void 0,
    organization_domain_common: void 0,
    organization_membership_quota_exceeded: void 0,
    organization_minimum_permissions_needed: void 0,
    passkey_already_exists: void 0,
    passkey_not_supported: void 0,
    passkey_pa_not_supported: void 0,
    passkey_registration_cancelled: void 0,
    passkey_retrieval_cancelled: void 0,
    passwordComplexity: {
      maximumLength: "mniej ni\u017C {{length}} znak\xF3w",
      minimumLength: "{{length}} lub wi\u0119cej znak\xF3w",
      requireLowercase: "ma\u0142\u0105 liter\u0119",
      requireNumbers: "cyfr\u0119",
      requireSpecialCharacter: "znak specjalny",
      requireUppercase: "wielk\u0105 liter\u0119",
      sentencePrefix: "Twoje has\u0142o musi zawiera\u0107"
    },
    phone_number_exists: "Numer telefonu jest ju\u017C zaj\u0119ty. Prosz\u0119 spr\xF3bowa\u0107 innego.",
    zxcvbn: {
      couldBeStronger: "Twoje has\u0142o jest odpowiednie, ale mog\u0142oby by\u0107 silniejsze. Spr\xF3buj doda\u0107 wi\u0119cej znak\xF3w.",
      goodPassword: "Twoje has\u0142o jest wystarczaj\u0105co silne.",
      notEnough: "Twoje has\u0142o jest zbyt s\u0142abe. Spr\xF3buj doda\u0107 wi\u0119cej znak\xF3w.",
      suggestions: {
        allUppercase: "Unikaj u\u017Cywania samych wielkich liter.",
        anotherWord: "Dodaj wi\u0119cej s\u0142\xF3w, kt\xF3re s\u0105 rzadsze.",
        associatedYears: "Unikaj lat zwi\u0105zanych z Tob\u0105.",
        capitalization: "U\u017Cywaj wielkich liter cz\u0119sciej.",
        dates: "Unikaj dat zwi\u0105zanych z Tob\u0105.",
        l33t: "Unikaj przewidywalnego zamieniania liter, takich jak '@' za 'a'.",
        longerKeyboardPattern: "U\u017Cywaj d\u0142ugich wzor\xF3w na klawiaturze, zmieniaj\u0105c kierunek pisania wielokrotnie.",
        noNeed: "Mo\u017Cesz tworzy\u0107 silne has\u0142a bez u\u017Cywania symboli, cyfr lub wielkich liter.",
        pwned: "Je\u017Celi u\u017Cywasz tego has\u0142a gdzie indziej, zmie\u0144 je jak najszybciej.",
        recentYears: "Unikaj ostatnich lat.",
        repeated: "Unikaj powtarzanych s\u0142\xF3w i znak\xF3w.",
        reverseWords: "Unikaj wpisywania popularnych s\u0142\xF3w od ty\u0142u.",
        sequences: "Unikaj popularnych kombinacji znak\xF3w.",
        useWords: "U\u017Cywaj wielu s\u0142\xF3w, ale unikaj popularnych fraz."
      },
      warnings: {
        common: "This is a commonly used password.",
        commonNames: "Common names and surnames are easy to guess.",
        dates: "Dates are easy to guess.",
        extendedRepeat: 'Repeated character patterns like "abcabcabc" are easy to guess.',
        keyPattern: "Short keyboard patterns are easy to guess.",
        namesByThemselves: "Single names or surnames are easy to guess.",
        pwned: "Your password was exposed by a data breach on the Internet.",
        recentYears: "Recent years are easy to guess.",
        sequences: 'Common character sequences like "abc" are easy to guess.',
        similarToCommon: "This is similar to a commonly used password.",
        simpleRepeat: 'Repeated characters like "aaa" are easy to guess.',
        straightRow: "Straight rows of keys on your keyboard are easy to guess.",
        topHundred: "This is a frequently used password.",
        topTen: "This is a heavily used password.",
        userInputs: "There should not be any personal or page related data.",
        wordByItself: "Single words are easy to guess."
      }
    }
  },
  userButton: {
    action__addAccount: "Dodaj konto",
    action__manageAccount: "Zarz\u0105dzaj kontem",
    action__signOut: "Wyloguj",
    action__signOutAll: "Wyloguj ze wszystkich kont"
  },
  userProfile: {
    backupCodePage: {
      actionLabel__copied: "Skopiowane!",
      actionLabel__copy: "Skopiuj wszystkie",
      actionLabel__download: "Pobierz .txt",
      actionLabel__print: "Drukuj",
      infoText1: "Kody zapasowe zostan\u0105 w\u0142\u0105czone dla tego konta.",
      infoText2: "Przechowuj kody zapasowe w tajemnicy i bezpiecznie. Mo\u017Cesz wygenerowa\u0107 nowe kody, je\u015Bli podejrzewasz, \u017Ce zosta\u0142y skompromitowane.",
      subtitle__codelist: "Przechowuj je bezpiecznie i zachowaj w tajemnicy.",
      successMessage: "Kody zapasowe s\u0105 teraz w\u0142\u0105czone. Mo\u017Cesz u\u017Cy\u0107 jednego z tych kod\xF3w do zalogowania si\u0119 na swoje konto, je\u015Bli utracisz dost\u0119p do urz\u0105dzenia uwierzytelniaj\u0105cego. Ka\u017Cdy kod mo\u017Cna u\u017Cy\u0107 tylko raz.",
      successSubtitle: "Mo\u017Cesz u\u017Cy\u0107 jednego z tych kod\xF3w do zalogowania si\u0119 na swoje konto, je\u015Bli utracisz dost\u0119p do urz\u0105dzenia uwierzytelniaj\u0105cego.",
      title: "Dodaj weryfikacj\u0119 kodem zapasowym",
      title__codelist: "Kody zapasowe"
    },
    connectedAccountPage: {
      formHint: "Wybierz dostawc\u0119, aby po\u0142\u0105czy\u0107 konto.",
      formHint__noAccounts: "Nie ma dost\u0119pnych zewn\u0119trznych dostawc\xF3w kont.",
      removeResource: {
        messageLine1: "{{identifier}} zostanie usuni\u0119te z tego konta.",
        messageLine2: "Nie b\u0119dziesz ju\u017C m\xF3g\u0142 korzysta\u0107 z tego po\u0142\u0105czonego konta i wszystkie zale\u017Cne funkcje przestan\u0105 dzia\u0142a\u0107.",
        successMessage: "{{connectedAccount}} zosta\u0142 usuni\u0119ty z Twojego konta.",
        title: "Usu\u0144 po\u0142\u0105czone konto"
      },
      socialButtonsBlockButton: "Po\u0142\u0105cz konto {{provider|titleize}}",
      successMessage: "Dostawca zosta\u0142 dodany do Twojego konta.",
      title: "Dodaj po\u0142\u0105czone konto"
    },
    deletePage: {
      actionDescription: 'Wpisz "Usu\u0144 konto" poni\u017Cej aby kontynuowa\u0107.',
      confirm: "Usu\u0144 konto",
      messageLine1: "Czy na pewno chcesz usun\u0105\u0107 to konto?",
      messageLine2: "Ta operacja jest nieodwracalna i nie mo\u017Cna jej cofn\u0105\u0107.",
      title: "Usu\u0144 konto"
    },
    emailAddressPage: {
      emailCode: {
        formHint: "E-mail zawieraj\u0105cy kod weryfikacyjny zostanie wys\u0142any na ten adres e-mail.",
        formSubtitle: "Wprowad\u017A kod weryfikacyjny wys\u0142any na adres {{identifier}}",
        formTitle: "Kod weryfikacyjny",
        resendButton: "Wy\u015Blij ponownie kod",
        successMessage: "Adres e-mail {{identifier}} zosta\u0142 dodany do twojego konta."
      },
      emailLink: {
        formHint: "E-mail zawieraj\u0105cy link weryfikacyjny zostanie wys\u0142any na ten adres e-mail.",
        formSubtitle: "Kliknij w link weryfikacyjny w e-mailu wys\u0142anym na adres {{identifier}}",
        formTitle: "Link weryfikacyjny",
        resendButton: "Wy\u015Blij ponownie link",
        successMessage: "Adres e-mail {{identifier}} zosta\u0142 dodany do twojego konta."
      },
      removeResource: {
        messageLine1: "{{identifier}} zostanie usuni\u0119ty z tego konta.",
        messageLine2: "Nie b\u0119dzie ju\u017C mo\u017Cliwe zalogowanie si\u0119 za pomoc\u0105 tego adresu e-mail.",
        successMessage: "{{emailAddress}} zosta\u0142 usuni\u0119ty z twojego konta.",
        title: "Usu\u0144 adres e-mail"
      },
      title: "Dodaj adres e-mail",
      verifyTitle: "Verify email address"
    },
    formButtonPrimary__add: "Dodaj",
    formButtonPrimary__continue: "Kontynuuj",
    formButtonPrimary__finish: "Zako\u0144cz",
    formButtonPrimary__remove: "Usu\u0144",
    formButtonPrimary__save: "Zapisz",
    formButtonReset: "Anuluj",
    mfaPage: {
      formHint: "Wybierz metod\u0119 dodania.",
      title: "Dodaj weryfikacj\u0119 dwuetapow\u0105"
    },
    mfaPhoneCodePage: {
      backButton: "U\u017Cyj istniej\u0105cego numeru",
      primaryButton__addPhoneNumber: "Dodaj numer telefonu",
      removeResource: {
        messageLine1: "{{identifier}} nie b\u0119dzie ju\u017C otrzymywa\u0142 kod\xF3w weryfikacyjnych podczas logowania.",
        messageLine2: "Twoje konto mo\u017Ce by\u0107 mniej bezpieczne. Czy na pewno chcesz kontynuowa\u0107?",
        successMessage: "Weryfikacja kodem SMS w dwustopniowym procesie uwierzytelniania zosta\u0142a usuni\u0119ta dla {{mfaPhoneCode}}",
        title: "Usu\u0144 dwustopniow\u0105 weryfikacj\u0119"
      },
      subtitle__availablePhoneNumbers: "Wybierz numer telefonu, aby zarejestrowa\u0107 weryfikacj\u0119 kodem SMS w dwustopniowym procesie uwierzytelniania.",
      subtitle__unavailablePhoneNumbers: "Brak dost\u0119pnych numer\xF3w telefon\xF3w do zarejestrowania weryfikacji kodem SMS w dwustopniowym procesie uwierzytelniania.",
      successMessage1: "When signing in, you will need to enter a verification code sent to this phone number as an additional step.",
      successMessage2: "Save these backup codes and store them somewhere safe. If you lose access to your authentication device, you can use backup codes to sign in.",
      successTitle: "SMS code verification enabled",
      title: "Dodaj weryfikacj\u0119 kodem SMS"
    },
    mfaTOTPPage: {
      authenticatorApp: {
        buttonAbleToScan__nonPrimary: "Zamiast tego zeskanuj kod QR",
        buttonUnableToScan__nonPrimary: "Nie mo\u017Cna zeskanowa\u0107 kodu QR?",
        infoText__ableToScan: "Ustaw now\u0105 metod\u0119 logowania w swojej aplikacji autentykacyjnej i zeskanuj nast\u0119puj\u0105cy kod QR, aby po\u0142\u0105czy\u0107 go z Twoim kontem.",
        infoText__unableToScan: "Ustaw now\u0105 metod\u0119 logowania w swojej aplikacji autentykacyjnej i wprowad\u017A poni\u017Cszy klucz.",
        inputLabel__unableToScan1: "Upewnij si\u0119, \u017Ce w\u0142\u0105czona jest opcja jednorazowe has\u0142a lub has\u0142a oparte na czasie, a nast\u0119pnie zako\u0144cz \u0142\u0105czenie konta.",
        inputLabel__unableToScan2: "Alternatywnie, je\u015Bli Twoja aplikacja autentykacyjna obs\u0142uguje URI TOTP, mo\u017Cesz r\xF3wnie\u017C skopiowa\u0107 pe\u0142ny URI."
      },
      removeResource: {
        messageLine1: "Kody weryfikacyjne z tej aplikacji autentykacyjnej nie b\u0119d\u0105 ju\u017C wymagane podczas logowania.",
        messageLine2: "Twoje konto mo\u017Ce by\u0107 mniej bezpieczne. Czy na pewno chcesz kontynuowa\u0107?",
        successMessage: "Weryfikacja dwuetapowa za pomoc\u0105 aplikacji autentykacyjnej zosta\u0142a usuni\u0119ta.",
        title: "Usu\u0144 weryfikacj\u0119 dwuetapow\u0105"
      },
      successMessage: "Weryfikacja dwuetapowa jest teraz w\u0142\u0105czona. Przy logowaniu b\u0119dziesz musia\u0142 wprowadzi\u0107 kod weryfikacyjny z tej aplikacji jako dodatkowy krok.",
      title: "Dodaj aplikacj\u0119 autentykacyjn\u0105",
      verifySubtitle: "Wprowad\u017A kod weryfikacyjny wygenerowany przez Twoj\u0105 aplikacj\u0119 autentykacyjn\u0105",
      verifyTitle: "Kod weryfikacyjny"
    },
    mobileButton__menu: "Menu",
    navbar: {
      account: "Profil",
      description: "Zarz\u0105dzaj danymi konta.",
      security: "Bezpiecze\u0144stwo",
      title: "Konto"
    },
    passkeyScreen: {
      removeResource: {
        messageLine1: void 0,
        title: void 0
      },
      subtitle__rename: void 0,
      title__rename: void 0
    },
    passwordPage: {
      checkboxInfoText__signOutOfOtherSessions: "Zaleca si\u0119 wylogowanie z innych urz\u0105dze\u0144, kt\xF3re mog\u0142y u\u017Cywa\u0107 starego has\u0142a.",
      readonly: "Your password can currently not be edited because you can sign in only via the enterprise connection.",
      successMessage__set: "Twoje has\u0142o zosta\u0142o ustawione.",
      successMessage__signOutOfOtherSessions: "Wylogowano z wszystkich innych urz\u0105dze\u0144.",
      successMessage__update: "Twoje has\u0142o zosta\u0142o zaktualizowane.",
      title__set: "Ustaw has\u0142o",
      title__update: "Zmie\u0144 has\u0142o"
    },
    phoneNumberPage: {
      infoText: "Wiadomo\u015B\u0107 tekstowa zawieraj\u0105ca link weryfikacyjny zostanie wys\u0142ana na ten numer telefonu.",
      removeResource: {
        messageLine1: "{{identifier}} zostanie usuni\u0119ty z tego konta.",
        messageLine2: "Nie b\u0119dzie ju\u017C mo\u017Cliwe zalogowanie si\u0119 za pomoc\u0105 tego numeru telefonu.",
        successMessage: "{{phoneNumber}} zosta\u0142 usuni\u0119ty z twojego konta.",
        title: "Usu\u0144 numer telefonu"
      },
      successMessage: "{{identifier}} zosta\u0142 dodany do twojego konta.",
      title: "Dodaj numer telefonu",
      verifySubtitle: "Wpisz kod weryfikacyjny wys\u0142any na {{identifier}}",
      verifyTitle: "Zweryfikuj numer telefonu"
    },
    profilePage: {
      fileDropAreaHint: "Prze\u015Blij zdj\u0119cie w formacie JPG, PNG, GIF lub WEBP mniejsze ni\u017C 10 MB",
      imageFormDestructiveActionSubtitle: "Usu\u0144 zdj\u0119cie",
      imageFormSubtitle: "Prze\u015Blij zdj\u0119cie",
      imageFormTitle: "Zdj\u0119cie profilowe",
      readonly: "Your profile information has been provided by the enterprise connection and cannot be edited.",
      successMessage: "Tw\xF3j profil zosta\u0142 zaktualizowany.",
      title: "Edytuj profil"
    },
    start: {
      activeDevicesSection: {
        destructiveAction: "Wyloguj z urz\u0105dzenia",
        title: "Aktywne urz\u0105dzenia"
      },
      connectedAccountsSection: {
        actionLabel__connectionFailed: "Spr\xF3buj ponownie",
        actionLabel__reauthorize: "Autoryzuj teraz",
        destructiveActionTitle: "Od\u0142\u0105cz",
        primaryButton: "Po\u0142\u0105cz konto",
        subtitle__disconnected: void 0,
        subtitle__reauthorize: "The required scopes have been updated, and you may be experiencing limited functionality. Please re-authorize this application to avoid any issues",
        title: "Po\u0142\u0105czone konta"
      },
      dangerSection: {
        deleteAccountButton: "Usu\u0144 konto",
        title: "Niebezpiecze\u0144stwo"
      },
      emailAddressesSection: {
        destructiveAction: "Usu\u0144 adres email",
        detailsAction__nonPrimary: "Ustaw jako g\u0142\xF3wny",
        detailsAction__primary: "Zako\u0144cz weryfikacj\u0119",
        detailsAction__unverified: "Zako\u0144cz weryfikacj\u0119",
        primaryButton: "Dodaj adres email",
        title: "Adresy email"
      },
      enterpriseAccountsSection: {
        title: "Enterprise accounts"
      },
      headerTitle__account: "Konto",
      headerTitle__security: "Bezpiecze\u0144stwo",
      mfaSection: {
        backupCodes: {
          actionLabel__regenerate: "Wygeneruj kody",
          headerTitle: "Kody zapasowe",
          subtitle__regenerate: "Otrzymaj nowy zestaw bezpiecznych kod\xF3w zapasowych. Poprzednie kody zapasowe zostan\u0105 usuni\u0119te i nie b\u0119d\u0105 dzia\u0142a\u0107.",
          title__regenerate: "Wygeneruj nowe kody zapasowe"
        },
        phoneCode: {
          actionLabel__setDefault: "Ustaw jako domy\u015Blny",
          destructiveActionLabel: "Usu\u0144 numer telefonu"
        },
        primaryButton: "Dodaj weryfikacj\u0119 dwuetapow\u0105",
        title: "Weryfikacja dwuetapowa",
        totp: {
          destructiveActionTitle: "Usu\u0144",
          headerTitle: "Aplikacja autoryzacyjna"
        }
      },
      passkeysSection: {
        menuAction__destructive: void 0,
        menuAction__rename: void 0,
        title: void 0
      },
      passwordSection: {
        primaryButton__setPassword: "Ustaw has\u0142o",
        primaryButton__updatePassword: "Zmie\u0144 has\u0142o",
        title: "Has\u0142o"
      },
      phoneNumbersSection: {
        destructiveAction: "Usu\u0144 numer telefonu",
        detailsAction__nonPrimary: "Ustaw jako g\u0142\xF3wny",
        detailsAction__primary: "Zako\u0144cz weryfikacj\u0119",
        detailsAction__unverified: "Zako\u0144cz weryfikacj\u0119",
        primaryButton: "Dodaj numer telefonu",
        title: "Numery telefon\xF3w"
      },
      profileSection: {
        primaryButton: void 0,
        title: "Profil"
      },
      usernameSection: {
        primaryButton__setUsername: "Ustaw nazw\u0119 u\u017Cytkownika",
        primaryButton__updateUsername: "Zmie\u0144 nazw\u0119 u\u017Cytkownika",
        title: "Nazwa u\u017Cytkownika"
      },
      web3WalletsSection: {
        destructiveAction: "Usu\u0144 portfel",
        primaryButton: "Portfele Web3",
        title: "Portfele Web3"
      }
    },
    usernamePage: {
      successMessage: "Twoja nazwa u\u017Cytkownika zosta\u0142a zaktualizowana.",
      title__set: "Zmie\u0144 nazw\u0119 u\u017Cytkownika",
      title__update: "Zmie\u0144 nazw\u0119 u\u017Cytkownika"
    },
    web3WalletPage: {
      removeResource: {
        messageLine1: "{{identifier}} zostanie usuni\u0119ty z tego konta.",
        messageLine2: "Nie b\u0119dziesz ju\u017C m\xF3g\u0142 si\u0119 zalogowa\u0107 za pomoc\u0105 tego portfela web3.",
        successMessage: "{{web3Wallet}} zosta\u0142 usuni\u0119ty z Twojego konta.",
        title: "Usu\u0144 portfel web3"
      },
      subtitle__availableWallets: "Wybierz portfel web3 do po\u0142\u0105czenia z Twoim kontem.",
      subtitle__unavailableWallets: "Nie ma dost\u0119pnych portfeli web3.",
      successMessage: "Portfel zosta\u0142 dodany do Twojego konta.",
      title: "Dodaj portfel web3",
      web3WalletButtonsBlockButton: void 0
    }
  },
  reverification: {
    alternativeMethods: {
      actionLink: "Uzyskaj pomoc",
      actionText: "Nie u\u017Cywasz \u017Cadnej z tych metod?",
      blockButton__backupCode: "U\u017Cyj kodu zapasowego",
      blockButton__emailCode: "Wy\u015Blij kod e-mailem do {{identifier}}",
      blockButton__password: "Zaloguj si\u0119 za pomoc\u0105 has\u0142a",
      blockButton__phoneCode: "Wy\u015Blij kod SMS-em do {{identifier}}",
      blockButton__totp: "U\u017Cyj aplikacji uwierzytelniaj\u0105cej",
      getHelp: {
        blockButton__emailSupport: "Skontaktuj si\u0119 z pomoc\u0105",
        content: "Je\u015Bli masz problem z weryfikacj\u0105 konta, wy\u015Blij do nas e-mail, a postaramy si\u0119 jak najszybciej przywr\xF3ci\u0107 dost\u0119p.",
        title: "Uzyskaj wsparcie"
      },
      subtitle: "Masz problem? Mo\u017Cesz u\u017Cy\u0107 dowolnej z tych metod weryfikacji.",
      title: "U\u017Cyj innej metody"
    },
    backupCodeMfa: {
      subtitle: "Tw\xF3j kod zapasowy to ten, kt\xF3ry otrzyma\u0142e\u015B podczas konfigurowania uwierzytelniania dwuetapowego.",
      title: "Wprowad\u017A kod zapasowy"
    },
    emailCode: {
      formTitle: "Kod weryfikacyjny",
      resendButton: "Nie otrzyma\u0142e\u015B kodu? Wy\u015Blij ponownie",
      subtitle: "aby kontynuowa\u0107 w {{applicationName}}",
      title: "Sprawd\u017A swoj\u0105 poczt\u0119 e-mail"
    },
    noAvailableMethods: {
      message: "Nie mo\u017Cna kontynuowa\u0107 weryfikacji. Brak dost\u0119pnych czynnik\xF3w uwierzytelniania.",
      subtitle: "Wyst\u0105pi\u0142 b\u0142\u0105d",
      title: "Nie mo\u017Cemy zweryfikowa\u0107 twojego konta"
    },
    password: {
      actionLink: "U\u017Cyj innej metody",
      subtitle: "Wprowad\u017A has\u0142o powi\u0105zane z twoim kontem",
      title: "Wprowad\u017A swoje has\u0142o"
    },
    phoneCode: {
      formTitle: "Kod weryfikacyjny",
      resendButton: "Nie otrzyma\u0142e\u015B kodu? Wy\u015Blij ponownie",
      subtitle: "aby kontynuowa\u0107 w {{applicationName}}",
      title: "Sprawd\u017A sw\xF3j telefon"
    },
    phoneCodeMfa: {
      formTitle: "Kod weryfikacyjny",
      resendButton: "Nie otrzyma\u0142e\u015B kodu? Wy\u015Blij ponownie",
      subtitle: "Aby kontynuowa\u0107, wprowad\u017A kod weryfikacyjny wys\u0142any na tw\xF3j telefon",
      title: "Sprawd\u017A sw\xF3j telefon"
    },
    totpMfa: {
      formTitle: "Kod weryfikacyjny",
      subtitle: "Aby kontynuowa\u0107, wprowad\u017A kod weryfikacyjny wygenerowany przez swoj\u0105 aplikacj\u0119 uwierzytelniaj\u0105c\u0105",
      title: "Weryfikacja dwuetapowa"
    }
  },
  waitlist: {
    start: {
      actionLink: void 0,
      actionText: void 0,
      formButton: void 0,
      subtitle: void 0,
      title: void 0
    },
    success: {
      message: void 0,
      subtitle: void 0,
      title: void 0
    }
  }
};
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  plPL
});
